client/components/credit-card-form-fields/index.jsx

```diff
@@ -390,4 +383,42 @@ export class CreditCardFormFields extends React.Component {
 	}
 }
 
+function CardholderNameField( { createField, shouldRenderCountrySpecificFields, autoFocus } ) {
+	const translate = useTranslate();
+	const { isStripeLoading, stripeLoadingError } = useStripe();
+
+	const disabled = isFieldDisabled( {
+		isStripeLoading,
+		stripeLoadingError,
+		isUsingEbanx: shouldRenderCountrySpecificFields(),
+	} );
+
+	return (
+		<>
+			{ createField( 'name', Input, {
+				disabled,
+				autoFocus,
+				label: translate( 'Cardholder Name {{span}}(as written on card){{/span}}', {
+					comment: 'Cardholder name label on credit card form',
+					components: {
+						span: <span className="credit-card-form-fields__explainer" />,
+					},
+				} ),
+				placeholder: ' ',
+			} ) }
+		</>
+	);
+}
+
+function isFieldDisabled( { isStripeLoading, stripeLoadingError, isUsingEbanx } ) {
+	const isStripeNotReady = isStripeLoading || stripeLoadingError;
+	if ( isUsingEbanx ) {
```

Comment:
For what it's worth, I'd find something like `return isStripeNotReady && ! isUsingEbanx` easier to follow here, but the logic looks good.